src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,119 @@
+.main {
+  display: grid;
+  justify-items: center;
+  justify-content: center;
+  grid-template-columns: repeat(4, 200px);
+  grid-template-rows: repeat(2, auto);
+  gap: 46px 48px;
+  padding: 50px 40px;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.card {
```

Comment:
The block class is called `card`, and the file is called `cards` - the name must be the same everywhere.